Description: Only include <glib.h> instead of other glib headers.
 Starting with glib 2.32 it is now mandatory to include <glib.h> instead
 of individual headers
Author: Michael Biebl <biebl@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=665556
Index: hal-0.5.14/hald/linux/addons/addon-acpi-buttons-toshiba.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-acpi-buttons-toshiba.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-acpi-buttons-toshiba.c	2012-03-25 00:46:54.750631674 +0100
@@ -40,7 +40,7 @@
 #include <string.h>
 #include <unistd.h>
 
-#include <glib/gmain.h>
+#include <glib.h>
 
 #include "libhal/libhal.h"
 #include "../../logger.h"
Index: hal-0.5.14/hald/linux/addons/addon-dell-backlight.cpp
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-dell-backlight.cpp	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-dell-backlight.cpp	2012-03-25 00:46:54.454631666 +0100
@@ -29,7 +29,7 @@
 
 #include <string.h>
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-generic-backlight.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-generic-backlight.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-generic-backlight.c	2012-03-25 00:46:54.358631663 +0100
@@ -35,7 +35,7 @@
 #include <sys/stat.h>
 #include <unistd.h> 
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-imac-backlight.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-imac-backlight.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-imac-backlight.c	2012-03-25 00:46:53.914631650 +0100
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 #include <sys/io.h>
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-input.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-input.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-input.c	2012-03-25 00:46:54.658631671 +0100
@@ -41,7 +41,7 @@
 #else
   #include <linux/input.h>
 #endif
-#include <glib/gmain.h>
+#include <glib.h>
 #include <glib/gprintf.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-ipw-killswitch.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-ipw-killswitch.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-ipw-killswitch.c	2012-03-25 00:46:54.066631655 +0100
@@ -36,7 +36,6 @@
 #include <unistd.h> 
 
 #include <glib.h>
-#include <glib/gmain.h>
 #include <glib/gstdio.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
Index: hal-0.5.14/hald/linux/addons/addon-leds.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-leds.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-leds.c	2012-03-25 00:46:54.842631677 +0100
@@ -36,7 +36,6 @@
 #include <unistd.h> 
 
 #include <glib.h>
-#include <glib/gmain.h>
 #include <glib/gstdio.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
Index: hal-0.5.14/hald/linux/addons/addon-macbookpro-backlight.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-macbookpro-backlight.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-macbookpro-backlight.c	2012-03-25 00:46:54.294631661 +0100
@@ -39,7 +39,7 @@
 #include <pci/pci.h>
 #include <unistd.h> 
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-omap-backlight.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-omap-backlight.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-omap-backlight.c	2012-03-25 00:46:54.114631656 +0100
@@ -41,7 +41,7 @@
 #include <fcntl.h>
 #include <unistd.h> 
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-rfkill-killswitch.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-rfkill-killswitch.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-rfkill-killswitch.c	2012-03-25 00:46:53.830631648 +0100
@@ -36,7 +36,6 @@
 #include <unistd.h> 
 
 #include <glib.h>
-#include <glib/gmain.h>
 #include <glib/gstdio.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
Index: hal-0.5.14/hald/linux/addons/addon-storage.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-storage.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-storage.c	2012-03-25 00:46:54.558631669 +0100
@@ -39,7 +39,7 @@
 #include <string.h>
 #include <sys/ioctl.h>
 #include <unistd.h>
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
Index: hal-0.5.14/hald/linux/addons/addon-usb-csr.c
===================================================================
--- hal-0.5.14.orig/hald/linux/addons/addon-usb-csr.c	2009-08-24 14:42:30.000000000 +0200
+++ hal-0.5.14/hald/linux/addons/addon-usb-csr.c	2012-03-25 00:46:54.194631658 +0100
@@ -29,7 +29,7 @@
 #include <string.h>
 #include <usb.h>
 
-#include <glib/gmain.h>
+#include <glib.h>
 #include <dbus/dbus-glib.h>
 #include <dbus/dbus-glib-lowlevel.h>
 
